### PR TITLE
Add information about not to perform the install step

### PR DESCRIPTION
**Pull request by larkox**
#### Summary
Add information about not to perform the install step

#### Ticket Link
None

### CODE REVIEW

File: docs/installation/zoom-configuration/zoom-setup-user-level-app.md [hunk 1]
```diff
@@ -39,6 +39,11 @@ You can set the **OAuth ClientID** and **OAuth Secret**, generated by Zoom, and
...
 ![Scopes screen](../../.gitbook/assets/screenshot-from-2020-06-05-19-37-47%20%282%29.png)
 
+## Do not perform the Install step
+
+* Zoom has one last option called Install. There is no need to perform this action.
+* If you perform this action, you will see an error on Mattermost. This is expected.
+
 ## Deauthorization
 
 This plugin allows users to be deauthorized directly from Zoom, in order to comply with Zoom’s commitment to security and the protection of user data. If you **would like to publish on Zoom Marketplace**, you will be able to set up a Deauthorization URL.
```
Comment (justinegeffen) [inline docs/installation/zoom-configuration/zoom-setup-user-level-app.md:45]:
```suggestion
This plugin allows users to be deauthorized directly from Zoom, in order to comply with Zoom’s commitment to security and the protection of user data. If you **would like to publish on Zoom Marketplace**, you can set up a deauthorization URL.
```

File: docs/installation/zoom-configuration/zoom-setup-oauth.md [hunk 2]
```diff
@@ -6,50 +6,53 @@ description: Steps to configure your Zoom account to work with Mattermost
...
 
 ![Scopes screen](../../.gitbook/assets/screenshot-from-2020-06-05-19-37-47%20%282%29.png)
 
+## Do not perform the install step
+
+Zoom has one last option called **Install**. There is no need to perform this action. However, if you perform this action inadvertently, you'll see an error on Mattermost. This is expected.
+
 ## Deauthorization
 
 This plugin allows users to be deauthorized directly from Zoom, in order to comply with Zoom’s commitment to security and the protection of user data. If you **would like to publish on Zoom Marketplace**, you will be able to set up a Deauthorization URL.
```
Comment (justinegeffen) [inline docs/installation/zoom-configuration/zoom-setup-oauth.md:46]:
```suggestion
This plugin allows users to be deauthorized directly from Zoom, in order to comply with Zoom’s commitment to security and the protection of user data. If you **would like to publish on Zoom Marketplace**, you can set up a deauthorization URL.
```

File: docs/installation/zoom-configuration/zoom-setup-user-level-app.md [hunk 3]
```diff
@@ -4,54 +4,54 @@ description: Steps to configure your Zoom account to work with Mattermost
...
+7. Select **Create**.
 
 ![Create an OAuth app screen](../../.gitbook/assets/screenshot-from-2020-06-05-19-31-06%20%282%29.png)
 
-## Configure Your New OAuth App to Work with Mattermost
+## Configure your new OAuth app to work with Mattermost
 
-* If you **would like to publish on Zoom Marketplace**, you will find two sets of values: **development** and **production**. Make sure you follow the next steps with the production values.
-* Go to the **App Credentials** tab on the left. Here you'll find your **Client ID** and **Client Secret**.
+1. If you **would like to publish on Zoom Marketplace**, you'll find two sets of values: **development** and **production**. Make sure you follow the next steps with the production values.
```
Comment (justinegeffen) [inline docs/installation/zoom-configuration/zoom-setup-user-level-app.md:25]:
```suggestion
If you **would like to publish on Zoom Marketplace**, you'll find two sets of values: **development** and **production**. Make sure you follow the next steps with the production values.
```

File: docs/installation/zoom-configuration/zoom-setup-user-level-app.md [hunk 4]
```diff
@@ -4,54 +4,54 @@ description: Steps to configure your Zoom account to work with Mattermost
...
 
 ![Create an OAuth app screen](../../.gitbook/assets/screenshot-from-2020-06-05-19-31-06%20%282%29.png)
 
-## Configure Your New OAuth App to Work with Mattermost
+## Configure your new OAuth app to work with Mattermost
 
-* If you **would like to publish on Zoom Marketplace**, you will find two sets of values: **development** and **production**. Make sure you follow the next steps with the production values.
-* Go to the **App Credentials** tab on the left. Here you'll find your **Client ID** and **Client Secret**.
+If you **would like to publish on Zoom Marketplace**, you'll find two sets of values: **development** and **production**. Make sure you follow the next steps with the production values.
+2. Go to the **App Credentials** tab on the left. Here you'll find your **Client ID** and **Client Secret**.
```
Comment (justinegeffen) [inline docs/installation/zoom-configuration/zoom-setup-user-level-app.md:26]:
```suggestion
1. Go to the **App Credentials** tab on the left. Here you'll find your **Client ID** and **Client Secret**.
```

File: docs/installation/zoom-configuration/zoom-setup-user-level-app.md [hunk 5]
```diff
@@ -4,54 +4,54 @@ description: Steps to configure your Zoom account to work with Mattermost
...
   * These will be needed during [Mattermost Setup]().
-* Enter a Valid **Redirect URL for OAuth** \(`https://SITEURL/plugins/zoom/oauth2/complete`\) and add the same URL under **Whitelist URL**.
+* Enter a valid **Redirect URL for OAuth** \(`https://SITEURL/plugins/zoom/oauth2/complete`\) and add the same URL under **Whitelist URL**.
```
Comment (justinegeffen) [inline docs/installation/zoom-configuration/zoom-setup-user-level-app.md:28]:
```suggestion
2. Enter a valid **Redirect URL for OAuth** \(`https://SITEURL/plugins/zoom/oauth2/complete`\) and add the same URL under **Whitelist URL**.
```

File: docs/installation/zoom-configuration/zoom-setup-user-level-app.md [hunk 6]
```diff
@@ -4,54 +4,54 @@ description: Steps to configure your Zoom account to work with Mattermost
...
+
 ## Deauthorization
 
-This plugin allows users to be deauthorized directly from Zoom, in order to comply with Zoom’s commitment to security and the protection of user data. If you **would like to publish on Zoom Marketplace**, you will be able to set up a Deauthorization URL.
+This plugin allows users to be deauthorized directly from Zoom, in order to comply with Zoom’s commitment to security and the protection of user data. If you **would like to publish on Zoom Marketplace**, you can set up a deauthorization URL.
 
-* Click on **Information**.
-* Near the end of the page, you will find a section called **Deauthorization Notification**.
-* Enter a valid **Endpoint URL** \(`https://SITEURL/plugins/zoom/deauthorization?secret=WEBHOOKSECRET`\). 
+1. Click on **Information**.
```
Comment (justinegeffen) [inline docs/installation/zoom-configuration/zoom-setup-user-level-app.md:46]:
```suggestion
1. Select **Information**.
```

File: docs/installation/zoom-configuration/zoom-setup-user-level-app.md [hunk 7]
```diff
@@ -4,54 +4,54 @@ description: Steps to configure your Zoom account to work with Mattermost
...
-* Enter a name for your app.
-* Choose **User-managed app** as the app type.
-* Choose whether you **Would like to publish this app on Zoom Marketplace**. In most cases you'll want this to be disabled, but the plugin support also apps to be published in the Zoom Marketplace.
-* Click **Create**.
+1. Go to [https://marketplace.zoom.us/](https://marketplace.zoom.us/) and log in as an admin.
+2. In the top left click on **Develop** and then **Build App**.
+3. Select **OAuth** in **Choose your app type** section.
+4. Enter a name for your app.
+5. Choose **User-managed app** as the app type.
+6. Choose whether you **Would like to publish this app on Zoom Marketplace**. In most cases you'll want this to be disabled, but the plugin support also apps to be published in the Zoom Marketplace.
```
Comment (justinegeffen) [inline docs/installation/zoom-configuration/zoom-setup-user-level-app.md:18]:
```suggestion
6. Choose whether you **Would like to publish this app on Zoom Marketplace**. In most cases you'll want this to be disabled, but the plugin supports apps that are published in the Zoom Marketplace.
```

File: docs/installation/zoom-configuration/zoom-setup-user-level-app.md [hunk 8]
```diff
@@ -4,54 +4,54 @@ description: Steps to configure your Zoom account to work with Mattermost
...
 
-* Click on **Scopes**.
-* Add the following scopes: **meeting:read**, **user:read**.
+Select **Scopes** and add the following scopes: **meeting:read**, **user:read**.
 
 ![Scopes screen](../../.gitbook/assets/screenshot-from-2020-06-05-19-37-47%20%282%29.png)
 
+## Do not perform the install step
+
+oom has one last option called **Install**. There is no need to perform this action. However, if you perform this action inadvertently, you'll see an error on Mattermost. This is expected.
```
Comment (justinegeffen) [inline docs/installation/zoom-configuration/zoom-setup-user-level-app.md:40]:
```suggestion
Zoom has one last option called **Install**. There is no need to perform this action. However, if you perform this action inadvertently, you'll see an error on Mattermost. This is expected.
```

File: docs/installation/zoom-configuration/zoom-setup-oauth.md [hunk 9]
```diff
@@ -6,50 +6,53 @@ description: Steps to configure your Zoom account to work with Mattermost
...
-* Enter a name for your app.
-* Choose **Account-level app** as the app type.
-* Choose whether you **Would like to publish this app on Zoom Marketplace**. In most cases you'll want this to be disabled, but the plugin support also apps to be published in the Zoom Marketplace.
-* Click **Create**.
+1. Go to [https://marketplace.zoom.us/](https://marketplace.zoom.us/) and log in as an admin.
+2. In the top left click on **Develop** and then **Build App**.
+3. Select **OAuth** in **Choose your app type** section.
+4. Enter a name for your app.
+5. Choose **Account-level app** as the app type.
+6. Choose whether you **Would like to publish this app on Zoom Marketplace**. In most cases you'll want this to be disabled, but the plugin support also apps to be published in the Zoom Marketplace.
```
Comment (justinegeffen) [inline docs/installation/zoom-configuration/zoom-setup-oauth.md:18]:
```suggestion
6. Choose whether you **Would like to publish this app on Zoom Marketplace**. In most cases you'll want this to be disabled, but the plugin supports apps that are published in the Zoom Marketplace. 
```

File: docs/installation/zoom-configuration/zoom-setup-oauth.md [hunk 10]
```diff
@@ -6,50 +6,53 @@ description: Steps to configure your Zoom account to work with Mattermost
...
 
-* Go to [https://marketplace.zoom.us/](https://marketplace.zoom.us/) and log in as an admin.
-* In the top left click on **Develop** and then **Build App**.
-* Select **OAuth** in **Choose your app type** section.
-* Enter a name for your app.
-* Choose **Account-level app** as the app type.
-* Choose whether you **Would like to publish this app on Zoom Marketplace**. In most cases you'll want this to be disabled, but the plugin support also apps to be published in the Zoom Marketplace.
-* Click **Create**.
+1. Go to [https://marketplace.zoom.us/](https://marketplace.zoom.us/) and log in as an admin.
+2. In the top left click on **Develop** and then **Build App**.
```
Comment (justinegeffen) [inline docs/installation/zoom-configuration/zoom-setup-oauth.md:14]:
```suggestion
2. In the top left select **Develop** and then **Build App**.
```

File: docs/installation/zoom-configuration/zoom-setup-user-level-app.md [hunk 11]
```diff
@@ -4,54 +4,54 @@ description: Steps to configure your Zoom account to work with Mattermost
...
 
-* Go to [https://marketplace.zoom.us/](https://marketplace.zoom.us/) and log in as an admin.
-* In the top left click on **Develop** and then **Build App**.
-* Select **OAuth** in **Choose your app type** section.
-* Enter a name for your app.
-* Choose **User-managed app** as the app type.
-* Choose whether you **Would like to publish this app on Zoom Marketplace**. In most cases you'll want this to be disabled, but the plugin support also apps to be published in the Zoom Marketplace.
-* Click **Create**.
+1. Go to [https://marketplace.zoom.us/](https://marketplace.zoom.us/) and log in as an admin.
+2. In the top left click on **Develop** and then **Build App**.
```
Comment (justinegeffen) [inline docs/installation/zoom-configuration/zoom-setup-user-level-app.md:14]:
```suggestion
2. In the top left select **Develop** and then **Build App**.
```

**Comment by justinegeffen** [comment]
Thanks, @larkox! This is a great addition and very helpful. I gave the whole document a review mainly for formatting and punctuation, but also made a few minor changes to the content for clarity. Let me know if you have any questions, otherwise I think this is good to go. :)

**Review by justinegeffen — APPROVED** [review]

**Comment by justinegeffen** [comment]
Sorry, one thing I noticed that I forgot to mention - the references to Mattermost setup are blank at the moment. Will they be added in a separate PR?

**Review by justinegeffen — APPROVED** [review]

**Comment by larkox** [comment]
There seem to be several problems with references, so I will fix them in a different PR.

**Pull request merged**